Client/Components/Compare.razor.cs

```diff
@@ -72,6 +72,7 @@ private async void UpdateHomeCountry(string name)
             _selectedHome = name;
             _country = await _apiHandler.FetchCountryByYearAsync(_httpClient, _availableCountries[name], _date);
             _dataMetrics = GetValidMetrics();
+            _availableMetrics = _dataMetrics;
```

Comment:
This might cause a problem since setting the variables to the same pointer makes them the same array. 
The problem will be with the text displayed when no data is selected.

Reply:
fixed